### PR TITLE
microarch manual edits

### PR DESCRIPTION
**Pull request by dpgrubb13**

### CODE REVIEW

File: docs/background.adoc [hunk 1]
```diff
@@ -24,13 +24,13 @@ image::diag/domains.png[DLP Domains,align=center,title-align=center]
 
 *HPC, scientific computing, and ML training workloads* are typically characterized by very long application vector lengths, exhibiting a high degree of data-level parallelism.
 Systems targeting these workloads are generally designed and deployed at datacenter scale, with available data-level-parallelism exploited across the many processors in a multi-node cluster.
-While single-core efficiency and performance remain important for this domain, scalability towards multi-core, multi-node systems is a more critical concern.
+While single-core efficiency and performance remain important for this domain, scalability towards multi-core, multi-node systems are of more critical concern.
```
Comment (jerryz123) [inline docs/background.adoc:27]: The subject of this sentence is singular - "scalability", so "is" should be correct here

File: docs/background.adoc [hunk 2]
```diff
@@ -102,10 +103,9 @@ Intel and AMD out-of-order cores support some form of the SSE or AVX extensions
 
 *VLIW ISAs with SIMD extensions* have traditionally dominated the space of DSP and similar domain-specialized cores, as well as embedded cores.
 For these deployment scenarios, power and area constraints preclude the deployment of costly microarchitectural features towards high instruction throughput in a general-purpose ISA, such as speculative execution and register renaming.
+The structure of DSP kernels presents a good fit for the VLIW paradigm, such as regular loops, rather than branch-dense control code, are easier to optimize on a VLIW microarchitecture.
```
Comment (jerryz123) [inline docs/background.adoc:106]: I think this reads clearer if "such as" were replaced with "since"

File: docs/execute.adoc [hunk 3]
```diff
@@ -37,10 +37,10 @@ image::diag/iqconfigs.png[Issue queue configurations,width=900,align=center,titl
...
+ * The *split* configuration features separate integer and floating-point sequencers each fed by an independent issue queue. This configuration is the most performant and the most flexible, as it can dynamically schedule out-of-order across integer and floating-point instructions. It incurs additional area costs owing to the separate floating-point issue queue.
  * The *multi-ALU* configuration adds an additional integer sequencer and functional units to enable dual-issue of integer ALU operations
- * The *multi-MAC* and *multi-FMA* configurations add additional integer or floating point sequencers along with functional units to enable dual-issue of integer or floating-point multiply-accumulate operations, respectively
+ * The *multi-MAC* and *multi-FMA* configurations add additional integer or floating point sequencers along with functional units to enable dual-issue of integer or floating-point multiply-accumulate operations, respectively. This creates an "unbalanced" machine with respect to the vector memory bandwidth that increases the maximum performance of the machine for kernels with sufficient reuse.
```
Comment (jerryz123) [inline docs/execute.adoc:43]: The comment on these configs being more suitable for higher-reuse kernels is good, but I think the term "unbalanced" has some negative connotations. Is there a better way to phrase this?

File: docs/execute.adoc [hunk 4]
```diff
@@ -111,14 +111,14 @@ Since Saturn is designed around `DLEN`-wide element groups as the base throughpu
 The scheduling mechanism precisely tracks which element groups an instruction or operation has yet to read-or-write to interlock the sequencers.
 
 In Saturn, the "out-of-order instruction window" includes all instructions in the issue queues (but not the VDQ), the instructions currently in progress within the sequencers, and any operations which have not yet completed execution in the functional unit datapaths.
-All instructions in this window must advertise a precise set of element groups they have not yet read or written, along with the age tag of the instruction.
+All instructions in this window must advertise a precise set of element groups that have not yet read or written, along with the age tag of the instruction.
```
Comment (jerryz123) [inline docs/execute.adoc:114]: Can you make this "groups that will be read or written by this instruction in future cycles"

File: docs/execute.adoc [hunk 5]
```diff
@@ -166,6 +166,7 @@ Functional units can either be pipelined or iterative.
 Pipelined functional units have fixed execution latencies, so once an operation is issued, it will execute without stalling.
 The sequencing mechanism checks for future write port conflicts on the target VRF bank across inflight and simultaneously sequenced operations to ensure that the next sequenced operation will not induce a structural hazard on the write port in the future.
 If a conflict is detected, the younger operation will be stalled and will likely start executing the very next cycle.
+This allows the operations to then proceed across the banks without any further conflicts with each other, barring other hazards.
```
Comment (jerryz123) [inline docs/execute.adoc:169]: This current phrasing is a bit confusing "without further conflicts, barring other hazards". What are you trying to emphasize here?

Comment (dpgrubb13) [reply docs/execute.adoc]: Rereading it, I think the comment isn't important. It was just meant to emphasize that the operations become offset and will proceed without conflicting with each other again unless some other action in the machine causes them to come into conflict again.

**Pull request merged**